src/cow-react/modules/trade/containers/TradeWidget/index.tsx

```diff
@@ -110,54 +76,70 @@ export function TradeWidget(props: TradeWidgetProps) {
   const throttledOnSwitchTokens = useThrottleFn(onSwitchTokens, 500)
 
   return (
-    <styledEl.Container>
+    <styledEl.Container id={id}>
       <styledEl.ContainerBox>
         <styledEl.Header>
           <TradeWidgetLinks />
-          {settingsWidget}
+          {!lockScreen && settingsWidget}
         </styledEl.Header>
 
-        <CurrencyInputPanel
-          id="input-currency-input"
-          disableNonToken={false}
-          chainId={chainId}
-          loading={currenciesLoadingInProgress}
-          onCurrencySelection={onCurrencySelection}
-          onUserInput={onUserInput}
-          allowsOffchainSigning={allowsOffchainSigning}
-          currencyInfo={inputCurrencyInfo}
-          showSetMax={showSetMax}
-          topLabel={inputCurrencyInfo.label}
-        />
-        {!isWrapOrUnwrap && middleContent}
-        <styledEl.CurrencySeparatorBox withRecipient={!isWrapOrUnwrap && showRecipient}>
-          <CurrencyArrowSeparator
-            isCollapsed={false}
-            onSwitchTokens={throttledOnSwitchTokens}
-            withRecipient={showRecipient}
-            isLoading={isTradePriceUpdating}
-            hasSeparatorLine={true}
-            border={true}
-          />
-        </styledEl.CurrencySeparatorBox>
-        <CurrencyInputPanel
-          id="output-currency-input"
-          disableNonToken={false}
-          chainId={chainId}
-          loading={currenciesLoadingInProgress}
-          isRateLoading={isRateLoading}
-          onCurrencySelection={onCurrencySelection}
-          onUserInput={onUserInput}
-          allowsOffchainSigning={allowsOffchainSigning}
-          currencyInfo={outputCurrencyInfo}
-          priceImpactParams={priceImpact}
-          topLabel={outputCurrencyInfo.label}
-        />
-        {showRecipient && (
-          <styledEl.StyledRemoveRecipient recipient={recipient || ''} onChangeRecipient={onChangeRecipient} />
+        {lockScreen ? (
+          lockScreen
+        ) : (
+          <>
+            <div>
+              <CurrencyInputPanel
+                id="input-currency-input"
```

Comment:
Would it be worth it to do it as a const, given is a param, and heavely used and repeated in the e2e?

Reply:
It's a good idea! Will add it in the next PR, don't you mind?

Reply:
is not too important. Fine if we don't do it actually. Soft suggestion